### ISSUE TITLE
cnvGWAS migration

### ISSUE THREAD

**Issue by lgeistlinger** [waldronlab/CNVRanger#27]
Hi @viniciushdasilva - I think it will be good to have a timeline for the migration. Will you be able to move things over until May 18th? That's the deadline for changes included in the upcoming Bioc3.13 release https://bioconductor.org/developers/release-schedule/? After that it will be another 6 months for the package being in the current state, so I think we should make an effort to meet the deadline.

**Comment by viniciushdasilva** [comment]
Agreed. I will merge a new `cnvGWAS` version until May 16th. Thus, we have time for some cosmetic improvements.

**Comment by viniciushdasilva** [comment]
Hi @lgeistlinger. I just updated a newer package version that performs linear regression without PLINK. Linear mixed models as present in `dev` branch will require further discussion as it depends on `lme4qtl`, which is not on CRAN.

**Comment by lgeistlinger** [comment]
Cool - thanks, Vinicius! I just pushed your changes on to Bioconductor.

> Linear mixed models as present in dev branch will require further discussion as it depends on lme4qtl, which is not on CRAN.

I think here your options reduce to (i) convince the `lme4qtl` authors to submit to CRAN (which I remember they were not reponsive to), or (ii) copy the corresponding part that you need from `lme4qtl` into CNVRanger.

**Comment by viniciushdasilva** [comment]
We could try to convince the authors again. But they refused last time we did.

**Comment by lgeistlinger** [comment]
How much copying of code would it involve do you think? I think they are pretty set on not submitting to CRAN / Bioc.

**Comment by viniciushdasilva** [comment]
`dev` branch needs core functions from `lme4qtl`. We could try to invite them and ask if we can copy the necessary code from `lme4qtl` into our package, at least until they manage to upload their package to a proper repository. Nevertheless, we could cite their publication in the `cnvGWAS` function help page and in our vignette explicitly.
